snmp/datadog_checks/snmp/data/conf.yaml.example

```diff
@@ -1,95 +1,100 @@
-## WARNING: This configuration files is for SNMP v1, v2 and v3
-## Check carefully all variables depending of your SNMP version.
-##
-## NOTE: See SNMP integration setup guide and examples at https://docs.datadoghq.com/network_monitoring/devices/setup
-
+## All options defined here are available to all instances.
```

Comment:
Could you make sure to keep the previous comments ?

```
## WARNING: This configuration files is for SNMP v1, v2 and v3
## Check carefully all variables depending of your SNMP version.
##
## NOTE: See SNMP integration setup guide and examples at https://docs.datadoghq.com/network_monitoring/devices/setup
```

Reply:
@matheuskja https://github.com/DataDog/integrations-core/blob/eed8b9c705724c67d43e1a179b09a05a5ace79e7/kafka_consumer/assets/configuration/spec.yaml#L5-L17